plugins/module_utils/network/iosxr/rm_templates/vrf.py

```diff
@@ -0,0 +1,487 @@
+                            },
+                        },
+                    },
+                },
+            },
+        },
+        {
+            "name": "evpn_route_sync",
+            "getval": re.compile(
+                r"""
+                ^vrf\s(?P<name>\S+)
+                \s+evpn-route-sync\s(?P<evpn_route_sync>\d+)
+                $""", re.VERBOSE,
+            ),
+            "setval": "vrf {{ name }} evpn-route-sync {{ evpn_route_sync }}",
+            "result": {
+                '{{ name }}': {
+                    'name': '{{ name }}',
+                    "evpn_route_sync": "{{ evpn_route_sync }}",
+                },
+            },
+        },
+        {
+            "name": "fallback_vrf",
+            "getval": re.compile(
+                r"""
+                ^vrf\s(?P<name>\S+)
+                \s+fallback-vrf\s\"(?P<fallback_vrf>\S+)\"
+                $""", re.VERBOSE,
+            ),
+            "setval": "vrf {{ name }} fallback-vrf {{ fallback_vrf }}",
+            "result": {
+                '{{ name }}': {
+                    'name': '{{ name }}',
+                    "fallback_vrf": "{{ fallback_vrf }}",
+                },
+            },
+        },
+        {
+            "name": "mhost_ipv4_default_interface",
+            "getval": re.compile(
+                r"""
+                ^vrf\s(?P<name>\S+)
+                \s+mhost
+                (\s(?P<afi>ipv4|ipv6))?
+                \s+default-interface\s(?P<ipv4_default_interface>\S+)
+                $""", re.VERBOSE,
+            ),
+            "setval": "vrf {{ name }} mhost ipv4 default-interface {{ mhost ipv4 default_interface }}",
+            "compval": "mhost.ipv4.default.interface",
+            "result": {
+                '{{ name }}': {
+                    'name': '{{ name }}',
+                    "mhost": {
+                        "afi": "{{ afi }}",
+                        "default_interface": "{{ ipv4_default_interface }}",
+                    },
+                },
+            },
+        },
+        {
+            "name": "rd",
+            "getval": re.compile(
+                r"""
+                ^vrf\s(?P<name>\S+)
+                \s+rd\s(?P<rd>\S+)
+                $""", re.VERBOSE,
+            ),
+            "setval": "vrf {{ name }} rd {{ rd }}",
+            "compval": "rd",
+            "result": {
+                '{{ name }}': {
+                    'name': '{{ name }}',
+                    "rd": "{{ rd }}",
+                },
+            },
+        },
+        {
+            "name": "remote_route_filtering_disable",
+            "getval": re.compile(
+                r"""
+                ^vrf\s(?P<name>\S+)
+                \s+remote-route-filtering\s(?P<disable>disable)
+                $""", re.VERBOSE,
+            ),
+            "setval": "vrf {{ name }} remote-route-filtering disable",
+            "compval": "remote_route_filtering.disable",
```

Comment:
Why do we need compval? Adding strict parsers eliminates the requirement of compval in most cases.

Reply:
keeping the compval for now